public/about.html

```
@@ -5,190 +5,108 @@
-        }
+  <div class="ornament" aria-hidden="true">
+    <svg viewBox="0 0 120 14" xmlns="http://www.w3.org/2000/svg" role="img" aria-hidden="true">
+      <path d="M6 7 C24 1, 36 13, 54 7 C72 1, 84 13, 102 7" stroke-linecap="round" stroke-linejoin="round"/>
+    </svg>
+  </div>
 
-        .about-content a:hover {
-            color: var(--accent-1);
-            text-shadow: 0 0 10px var(--glow-1);
-        }
+  <p>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </p> ], start tag match failed [ <p> ] on line 76.

(tag-pair)